pkg/pdf/pdf.go

```diff
@@ -481,6 +482,24 @@ func (s *PdfMaroto) Barcode(code string, prop ...props.Barcode) (err error) {
 	return
 }
 
+// DataMatrixCode creates an datamatrix code inside a cell.
+func (s *PdfMaroto) DataMatrixCode(code string, prop ...props.Rect) {
```

Comment:
Is missing some unit tests to this method.

Reply:
should be solved by 7ec9688